pack/tde/sfk_encounter.json

```diff
@@ -0,0 +1,873 @@
+		"clues": null,
+		"code": "06126",
+		"double_sided": true,
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 8,
+		"faction_code": "mythos",
+		"flavor": "Much of the valley realm of Ooth-Nargai, ruled by the dreamer Kuranes, eludes the grasp of time. Nothing here ages or falls into disrepair. It is a realm of wonders and marvels - how many years might pass here before you even noticed?",
+		"illustrator": "Julian Kok",
+		"name": "The King's Decree",
+		"pack_code": "sfk",
+		"position": 126,
+		"quantity": 1,
+		"stage": 2,
+		"text": "<b>Objective</b> - Find and uncover as many Signs of the Gods in this region as you can. At the end of the round, if each investigator is at the same [[Port]] location, you may move to another region by advancing this act.\n<b>Objective</b> - If the investigators have uncovered 10 Signs of the Gods, proceed to <b>(→R1)</b>.",
+		"type_code": "act"
+	},
+	{
+		"clues": 1,
+		"code": "06127",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 9,
+		"faction_code": "mythos",
+		"flavor": "\"Very pleasant were the suburbs of Ulthar, with their little green cottages and neatly fenced farms; and still pleasanter was the quaint town itself...\"\n – H. P. Lovecraft, <u>The Dream-Quest of Unknown Kadath</u>",
+		"illustrator": "Frej Agelii",
+		"name": "Ulthar",
+		"pack_code": "sfk",
+		"position": 127,
+		"quantity": 1,
+		"shroud": 1,
+		"text": "Veiled.",
+		"traits": "Skai. City.",
+		"type_code": "location"
+	},
+	{
+		"back_link": "06127",
+		"code": "06127b",
+		"faction_code": "mythos",
+		"flavor": "As you walk the streets of the town, a horde of cats slowly congregates around you, following you everywhere you go.",
+		"hidden": true,
+		"name": "Cryptic Souls",
+		"pack_code": "sfk",
+		"position": 127,
+		"quantity": 1,
+		"text": "<b><hr>Check Campaign Log. </b>[[If the cats collected their tribute from the Zoogs]]<b>, read the following:</b>\n<blockquote><i>One by one, the cats demand a tribute in the form of physical affection. Despite the urgency of your task, they rub against your legs, leap onto your lap, and do not leave until they are thoroughly adored.</i></blockquote>\nLose all of your remaining actions and end your turn.\n<hr>[[If the investigators forced their way into the temple]]<b>, read the following:</b>\n<blockquote><i>The next thing you know, the cats have you completely surrounded. They hiss and growl, closing in from all sides.</i></blockquote>\nSpawn the Cats of Ulthar enemy engaged with you.\n<hr><b>Regardless:</b>\nFlip this card back over.",
```

Comment:
Please can you change using [[]] here to bold & italic tags, as in future we may search / change Traits in [[]] tags. Comes up twice in the line.

Reply:
Fixed!